test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -1664,6 +1664,14 @@ static int create_tm_system(void)
 		return -1;
 	}
 
+	/* Start TM system */
+	CU_ASSERT((rc = odp_tm_start(odp_tm)) == 0);
```

Comment:
The validation test now incorrectly assumes that dynamic updates are always supported as the TM is not stopped before dynamic updates in the test cases such as traffic_mngr_test_shaper().

The test should do stop-start around modifications either always or depending on the dynamic update capability.